datasets/fraud use case/README.md

```diff
@@ -0,0 +1,28 @@
+
+## Introduction to COOL
+
+![ Fraud Actions](./img/ato.png)
+
+### Background:
+
+User’s account may be taken over by some cyber attackers. To save time and resource, one attacker may steal more than one account within a short time. We can find such stolen accounts by analysing their actions.
+
+### User selection:
+
+Normal accounts seldom change the password. If a group of users who update password using the same IP in the same day, they can be treated as a cohort. It is likely that the accounts may be stolen by the same hackers.
+
+### Birth Criteria:
+
+Users who have update password using the same IP in the same day with others. Group by: IP + action date
+
+### Cohort Matrix:
+
```

Comment:
I think that we cannot record the ip and the device of the login action. We can only record the times of login. So how can we implement it?